meson.build

```diff
@@ -611,6 +620,13 @@ ui_sources = []
 subdir('po')
 subdir('plugins')
 
+conditional_unit_test_deps = []
+if conf_data.get('ENABLE_UNIT_TESTS', 0) == 1
+    gtest_proj = subproject('gtest')
```

Comment:
Is it possible to use system gtest? Like in https://github.com/mesonbuild/meson/blob/master/test%20cases/frameworks/2%20gtest/meson.build

Reply:
Added a fallback mechanism to start with system gtest and fallback to the subproject.